test/files/jvm/varargs-separate-bytecode/Props_1.flags

```diff
@@ -0,0 +1 @@
+-sourcepath test/files/jvm/varargs-separate-bytecode/
```

Comment:
I haven't seen this done before in partest. What's the intent? Is there a simpler way?